l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -697,7 +697,7 @@ def setDettaglioLinee(self, cr, uid, invoice, body, context=None):
             prezzo_unitario = self._get_prezzo_unitario(cr, uid, line)
             DettaglioLinea = DettaglioLineeType(
                 NumeroLinea=str(line_no),
-                Descrizione=line.name.replace('\n', ' '),
+                Descrizione=unidecode(line.name.replace('\n', ' ')),
```

Comment:
nelle altre versioni ho proposto questa modifica #720 se vogliamo stare in linea